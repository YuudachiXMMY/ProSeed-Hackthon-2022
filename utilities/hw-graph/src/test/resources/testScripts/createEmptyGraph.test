# Test to create an empty graph

# Create a graph
CreateGraph Graph0

# Print all nodes in graph
ListNodes Graph0