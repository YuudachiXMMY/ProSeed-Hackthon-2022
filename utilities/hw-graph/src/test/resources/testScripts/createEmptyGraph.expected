# Test to create an empty graph

# Create a graph
created graph Graph0

# Print all nodes in graph
Graph0 contains: